aws-android-sdk-mobile-client/src/main/java/com/amazonaws/mobile/client/AWSMobileClient.java

```diff
@@ -1304,11 +1314,21 @@ public SignInResult confirmSignIn(final String signInChallengeResponse,
                                       final Map<String, String> clientMetadata) throws Exception {
 
         final InternalCallback<SignInResult> internalCallback = new InternalCallback<SignInResult>();
-        return internalCallback.await(_confirmSignIn(signInChallengeResponse, clientMetadata, internalCallback));
+        return internalCallback.await(_confirmSignIn(signInChallengeResponse, clientMetadata, Collections.<String, String>emptyMap(), internalCallback));
```

Comment:
nit: reduce redundancy
```suggestion
        return confirmSignIn(signInChallengeResponse, clientMetadata, Collections.<String, String>emptyMap());
```

Reply:
Same as above.